### PR TITLE
Removed 'using namespace' directive from public header

### PR DESCRIPTION
**Pull request by LarryOsterman**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [X] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [X] Doxygen docs
- [ ] Unit tests
- [X] No unwanted commits/changes
- [X] Descriptive title/description
  - [X] PR is single purpose
  - [ ] Related issue listed
- [X] Comments in source
- [X] No typos
- [ ] Update changelog
- [X] Not work-in-progress
- [ ] External references or docs updated
- [X] Self review of PR done
- [X] Any breaking changes?

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp [hunk 1]
```diff
@@ -67,7 +65,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Administra
      *
      * @returns Response containing the new updated setting.
      */
-    Azure::Response<Setting> UpdateSetting(
+    Azure::Response<Azure::Security::KeyVault::Administration::Models::Setting> UpdateSetting(
```
Comment (antkmsft) [inline sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp:68]:
```suggestion
    Azure::Response<Models::Setting> UpdateSetting(
```

File: sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp [hunk 2]
```diff
@@ -80,7 +78,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Administra
      *
      * @returns response containing the setting.
      */
-    Azure::Response<Setting> GetSetting(
+    Azure::Response<Azure::Security::KeyVault::Administration::Models::Setting> GetSetting(
```
Comment (antkmsft) [inline sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp:81]:
```suggestion
    Azure::Response<Models::Setting> GetSetting(
```

File: sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp [hunk 3]
```diff
@@ -91,11 +89,13 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Administra
      *
      * @returns Response containing a list of settings.
      */
-    Azure::Response<SettingsListResult> GetSettings(
+    Azure::Response<Azure::Security::KeyVault::Administration::Models::SettingsListResult>
+    GetSettings(
```
Comment (antkmsft) [inline sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp:93]:
```suggestion
    Azure::Response<Models::SettingsListResult> GetSettings(
```

File: sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp [hunk 4]
```diff
@@ -91,11 +89,13 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Administra
...
         const Azure::Core::Context& context = Azure::Core::Context{}) const;
 
   private:
-    Setting ParseSetting(std::vector<uint8_t> const& responseBody) const;
+    Azure::Security::KeyVault::Administration::Models::Setting ParseSetting(
+        std::vector<uint8_t> const& responseBody) const;
```
Comment (antkmsft) [inline sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp:98]:
```suggestion
    Models::Setting ParseSetting(std::vector<uint8_t> const& responseBody) const;
```

**Review by antkmsft — APPROVED** [review]

File: sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp [hunk 5]
```diff
@@ -91,11 +89,12 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Administra
      *
      * @returns Response containing a list of settings.
      */
-    Azure::Response<SettingsListResult> GetSettings(
+    Azure::Response<Models::SettingsListResult>
+    GetSettings(
```
Comment (antkmsft) [inline sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp:93]: This one is probably needed as well - I think clang format will fail if it is 2 lines:
```suggestion
    Azure::Response<Models::SettingsListResult> GetSettings(
```

File: sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp [hunk 6]
```diff
@@ -17,8 +17,6 @@
 #include <memory>
 #include <string>
 
-using namespace Azure::Security::KeyVault::Administration::Models;
```
[the remark this answers is not in the document]
Comment (ahsonkhan) [reply sdk/keyvault/azure-security-keyvault-administration/inc/azure/keyvault/administration/settings_client.hpp:20]: https://github.com/Azure/azure-sdk/issues/5888

**Pull request merged**